### DISCUSSION TITLE
Auto should allow none to signfy changes that don't impact the application versioning

### DISCUSSION THREAD

**Discussion by sheldonhull** [miniscruff/changie#522] [Ideas] [answered]
I know the auto versioning takes the least of the changes, so mostly this is a non-issue. However, if the CI or build scripts change, perhaps the versioning of the app could be left alone with a `auto: none` to signify no version change? 

Maybe it doesn't make sense, and I'll keep using patch, but was thinking about changes related to local development/build scripts and more and wondering if incrementing the patch on that type of stuff makes since at all. 

Open for discussion!

**Comment by miniscruff** [comment, marked as answer]
To me you wouldn't make a release at all if users are not affected. So I wouldn't do anything at all and just wait until changes do come in and then release.

Edit: One example is changie itself, I always maintain dependencies and sometimes update some tooling or configurations. Refactoring is another one that is not released or mentioned, such as this PR that rewrote all my internal uses of the cobra CLI and removed afero https://github.com/miniscruff/changie/pull/466 but I didn't release after it or make a changelog for it. 

With that said, I am not entirely sure what None would do? If the version doesn't change then changie doesn't do anything by design.

**Reply by sheldonhull** [reply]
I was thinking it would log changes related to development but I think I’m conflating git log and changes intent in this question. Devs have often been the audience of my chances and communicating changes to the project through changie helped me. Maybe I’ll use the new project features wip for that sometime instead.
